README.md

```diff
@@ -0,0 +1,53 @@
+# Houses For Sale Searcher - CodersCamp 2021/22 Projekt React
+## I. Zespół projektowy
+
+Zespół pracował w ramach kursu [CodersCamp](https://coderscamp.pl/).
+Aplikację wykonali uczestnicy kursu przy pomocy mentora.
+
+**Mentor**: [Filip Hałoń](https://github.com/FilipHalon)
+
+**Uczestnicy**:
+
+-   [Marta Goniszewska](https://github.com/mgoniszewska)
+-   [Tomasz Ikert](https://github.com/ike-tom)
+-   [Patryk Jurczyk](https://github.com/PatrykJurczyk)
+-   [Wojciech Okularczyk](https://github.com/shadowas-py)
+
+## II. [DEMO]()
+
+## III. Wykorzystane technologie:
+- React
+- JavaScript
+- HTML
+- CSS - SCSS
+
+## IV. Narzędzia z jakimi pracowaliśmy:
+- Firebase
+- Jira
+- Figma
+- Visual Studio Code
+- Git
+
+### [Design](https://www.figma.com/file/ZkiIq0vudllZEto6pU2UQh/Untitled?node-id=5%3A25)
+
+## V. Organizacja pracy
+
+Organizacja pracy zespołu oraz code review były przeprowadzane w serwisie GitHub.
+Zadania opisano oraz rozdzielano w oprogramowaniu Jira.
+Do daily korzystaliśmy z Google Meet. Daily były przeprowadzane w każdy wtorek, czwartek oraz niedzielę.
+Podczas codziennej komunikacji korzystano z Discorda.
+
+## VI. Uruchomienie projektu
+Do uruchomienia projektu na maszynie lokalnej należy wpisać w terminalu:
+```bash
+git clone git@github.com:team-prstmw/HousesForSaleSearcher.git
+```
+Następnie, aby zainstalować moduły, należy wpisac:
+```bash
+yarn install
+```
+Następnie:
+```bash
+npm run dev
```

Comment:
I guess the instruction could stick to recommending using `yarn` instead of `npm`.